ranger/data/scope.sh

```diff
@@ -160,7 +160,26 @@ handle_image() {
         #     exit 1;;
 
         ## PDF
-        # application/pdf)
+        application/pdf)
+            if [[ "$(uname)" = "Darwin" ]]; then
```

Comment:
I'm not sure we should check the uname here, if the binary's available then we can call it. If it's not the `exit 1` takes care of it.

Reply:
Okay. I will change accordingly.

Reply:
Thank you for bearing with us btw, I will try to respond faster.